sceptre/cli/drift.py

```diff
@@ -0,0 +1,82 @@
+import click
+
+from sceptre.context import SceptreContext
+from sceptre.cli.helpers import (
+    catch_exceptions,
+    write
+)
+from sceptre.plan.plan import SceptrePlan
+
+
+@click.group(name="drift")
+def drift_group():
+    """
+    Commands for calling drift detection.
+    """
+    pass
+
+
+@drift_group.command(name="detect", short_help="Run detect stack drift on running stacks.")
+@click.argument("path")
+@click.pass_context
+@catch_exceptions
+def drift_detect(ctx, path):
+    """
+    Detect stack drift and return if drift has occurred.
+    \f
```

Comment:
What's the purpose of this `\f`?

Reply:
I'm not sure. Something used by the docs generation I suppose? But all of them have this `\f`.